src/Http/Cookie/CookieManager.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Http\Cookie;
+
+use DateTimeImmutable;
+use Tempest\Clock\Clock;
+
+final class CookieManager
```

Comment:
I wish we had a better name for this, but I'm also not sure what that would be. 😛

Reply:
`Bakery`